xarray/coding/cftime_offsets.py

```diff
@@ -1139,7 +1219,7 @@ def date_range(
                 tz=tz,
                 normalize=normalize,
                 name=name,
-                closed=closed,
+                inclusive=inclusive,
```

Comment:
This requires pandas version 1.4 or greater, but our current minimum version is 1.3.  I'm not sure if we might be open to bumping that in #7461 as well ([pandas 1.4 was released 12 months ago today on 2022-01-22](https://github.com/pandas-dev/pandas/releases/tag/v1.4.0)).

Reply:
since our min-deps policy allows it and it actually reduces the maintenance burden, I'd say let's bump it.